backend/src/main/resources/db/changelog/sql/20250217095500-prize_story_name.sql

```diff
@@ -0,0 +1,10 @@
+--liquibase formatted sql
+
+--changeset andrea.rubino:20250217095500-1
+--comment: Add column story_name
+--preconditions onFail:MARK_RAN onError:HALT
+--precondition-sql-check expectedResult:0 select count(*) from information_schema.columns where table_name='prize' and column_name='story_name';
+
+ALTER TABLE prize
+    ADD COLUMN story_name TEXT NULL;
```

Comment:
It should not be null, We have to add a NOT NULL constraint and set 'overmind' as default value